bakery/env/staging.json

```diff
@@ -5,12 +5,14 @@
   "COPS_BUCKET_SAK_SECRET_NAME": "((aws-sandbox-secret-access-key))",
   "COPS_ARTIFACTS_S3_BUCKET": "openstax-sandbox-cops-artifacts",
   "COPS_CLOUDFRONT_URL": "https://cops.sandbox.openstax.org",
-  "DIST_FEED_FILE_URL": "https://raw.githubusercontent.com/openstax/content-manager-approved-books/master/approved-books.json",
+  "DIST_ARCHIVE_FEED_FILE_URL": "https://raw.githubusercontent.com/openstax/content-manager-approved-books/master/approved-books.json",
+  "DIST_GIT_FEED_FILE_URL": "https://raw.githubusercontent.com/openstax/content-manager-approved-books/bn/cnx%231234/abl-for-git-pipeline/approved-git-books.json",
```

Comment:
This may not be the file we want to check forever :laughing:

Reply:
I also notice there is no schema in place for this feed file. We should create one.